### PR TITLE
Allow upgrades from 4.4 and 4.3

### PR DESCRIPTION
**Pull request by asalkeld**
```release-note
Allow upgrades from v4
```
/cc @mjudeikis 
/cc @ehashman

### CODE REVIEW

File: pkg/plugin/plugin.go [hunk 1]
```diff
@@ -212,6 +212,11 @@ func (p *plugin) createOrUpdateExt(ctx context.Context, cs *api.OpenShiftManaged
 		isUpdate = false
 	}
 
+	switch cs.Config.PluginVersion {
+	case "v4.4", "v4.3":
+		backupEtcd = false
```
Comment (asalkeld) [inline pkg/plugin/plugin.go]: the backup didn't seem to work, so disabling..

**Comment by mjudeikis** [comment]
This is not enough. Something is off here as after upgrade it still runs with v4.4 images

**Comment by jim-minter** [comment]
I am pretty convinced we must not do this.  Closing.  If I'm absolutely wrong it can be re-opened.

**Pull request closed without merging**